framework/src/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -113,6 +117,12 @@
     public CFAbstractTransfer(CFAbstractAnalysis<V, S, T> analysis) {
         this.analysis = analysis;
         this.sequentialSemantics = !analysis.checker.hasOption("concurrentSemantics");
+        useJaifInference = analysis.getTypeFactory().getProcessingEnv().
+                getOptions().containsKey("useJaifInference");
```

Comment:
Renamed to performWholeProgramInference. We could also call it performInterProceduralInference. Please, let me know what your thoughts here.